frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/FetchAllRemotesAction.java

```diff
@@ -32,8 +32,12 @@ public LambdaLogger log() {
   protected void onUpdate(AnActionEvent anActionEvent) {
     super.onUpdate(anActionEvent);
 
-    val project = getProject(anActionEvent);
     val presentation = anActionEvent.getPresentation();
+    if (!presentation.isEnabled()) {
```

Comment:
I am not sure about it 🤔 maybe leave these changes to 3.3.1? 

Here I am thinking about a scenario when the action is visible, not enabled but we still want to provide some description (e.g. why the action is disabled) - we need to examine these changes carefully and see if the proper messages still appear when they should

Reply:
Ok, 3.3.1 FTW